source/brahma.c

```diff
@@ -261,7 +267,7 @@ s32 map_arm11_payload (void) {
 	dst = (void *)(g_expdata.va_exc_handler_base_W + OFFS_EXC_HANDLER_UNUSED);
 	size = (u8 *)&arm11_end - (u8 *)&arm11_start;
 
-	// TODO: sanitize 'size' 
+	// TODO: sanitize 'size'
 	if (size) {
 		memcpy(dst, src, size);
```

Comment:
One way to maintain volatile qualifier here would be to cast dst to (void *) in memcpy operations. However, I think the right thing to do here would be to roll out a custom version of memcpy.